lib/debug.js

```diff
@@ -185,11 +185,11 @@ var Debug = Class.$extend({
         if (!this.$data._caughtMessages[type][id]) {
             this.$data._caughtMessages[type][id] = true;
 
-            Sentry.captureException(error, {
-                level: type,
-                logger: "obsidian/debug",
-                tags: tags,
-                extra: extra
+            Sentry.configureScope((scope) => {
```

Comment:
use withScope instead

Reply:
tested with "withScope": https://sentry.io/organizations/wanadev-kz/issues/1272080092/?project=1380986&query=is%3Aunresolved